.github/workflows/release.yml

```diff
@@ -78,11 +79,18 @@ jobs:
 
           nextTag=''
           releaseType=${{ inputs.releaseType }}
-          if [ $releaseType == "minor" ]; then
-            # increment minor version and reset patch version
-            nextTag=$(echo "${currentTag}" | awk -F. '{OFS="."; $2+=1; $3=0; print $0}')
+          if [ $releaseType == "major" ]; then
+              # PBS-GO skipped the v1.0.0 major release - https://github.com/prebid/prebid-server/issues/3068
+              # If the current tag is v0.x.x, the script sets the next release tag to v2.0.0
+              # Otherwise, the script increments the major version by 1 and sets the minor and patch versions to zero
+              # For example, v2.x.x will be incremented to v3.0.0
```

Comment:
Excellent! I wasn't thinking of the 1.0 skip. Thank you for taking that into account.